java/test/jmri/configurexml/LoadAndStoreTestBase.java

```diff
@@ -160,29 +169,40 @@ static void loadFile(File inFile) throws Exception {
         new jmri.jmrit.catalog.configurexml.DefaultCatalogTreeManagerXml().readCatalogTrees();
     }
 
-    static File storeFile(File inFile) throws Exception {
+    // removed "static" here so we can access saveType field
+    private File storeFile(File inFile) throws Exception {
         String name = inFile.getName();
         FileUtil.createDirectory(FileUtil.getUserFilesPath() + "temp");
         File outFile = new File(FileUtil.getUserFilesPath() + "temp/" + name);
-        InstanceManager.getDefault(ConfigureManager.class).storeConfig(outFile);
+
+        // "Prefs", "UserPrefs", "Config", "User" or "All"
+        if (saveType.equals("Prefs")) {
+            InstanceManager.getDefault(ConfigureManager.class).storePrefs(outFile);
+        } else if (saveType.equals("UserPrefs")) {
+            InstanceManager.getDefault(ConfigureManager.class).storeUserPrefs(outFile);
+        } else if (saveType.equals("Config")) {
+            InstanceManager.getDefault(ConfigureManager.class).storeConfig(outFile);
+        } else if (saveType.equals("User")) {
+            InstanceManager.getDefault(ConfigureManager.class).storeUser(outFile);
+        } else if (saveType.equals("All")) {
+            InstanceManager.getDefault(ConfigureManager.class).storeAll(outFile);
+        } else {
+            log.error("Unknown save type {}.", saveType);
+        }
+
         return outFile;
     }
 
     @Test
     public void loadLoadStoreFileCheck() throws Exception {
+        Assume.assumeFalse(GraphicsEnvironment.isHeadless());
```

Comment:
Panel (subclass) files load GUI and will fail if headless… wasn't sure best way to do this. Since I have the saveType I could "Assume.assumeTrue/False(isHeadless())" based on that… Other than SaveType == All which others won't run headless?